### ISSUE TITLE
Is broadcast support possible

### ISSUE THREAD

**Issue by danjenkins** [balloob/pychromecast#213]
Is broadcast support possible? Wouldn't it be lovely to be able to broadcast those TTS messages in Home Assistant instead of playing them as media with the blip sound? I also presume (maybe wrongly) that when playing media (like music) and theres a broadcast message, the google home will go back to playing the media? 

Maybe broadcasting isnt even part of the chromecast protocol? But thought I'd ask!

**Comment by danjenkins** [comment]
Ah I think this is actually a Google Assistant feature. But I'd presume it uses the cast protocol to actually deliver the media?

**Comment by OttoWinter** [comment]
Yes this is indeed a Google Assistant feature.

> But I'd presume it uses the cast protocol to actually deliver the media?

No, the Assistant devices keep an active connection to Google's servers at all times. Looking at some packet captures I just took, this encrypted connection is also used for broadcasts.

> the google home will go back to playing the media?

I've also tried a tried to find a solution to this annoying problem, but haven't found any yet. Let me know if you find a way!
